src/main/java/teammates/common/util/Version.java

```diff
@@ -0,0 +1,167 @@
+package teammates.common.util;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.List;
+
+import com.google.appengine.api.modules.ModulesService;
+import com.google.appengine.api.modules.ModulesServiceFactory;
+
+public class Version {
+    private static ModulesService modulesService = ModulesServiceFactory.getModulesService();
+    private Integer major;  //  the first number
+    private Integer minor;  //  the second number
+    private String patch;   //  the rest of version
```

Comment:
Yes, to cover the case when more than 3 numbers may be used to present version. For example 11.05.01.01, etc.